src/packages/cli/src/__tests__/integrate.mysql.test.ts

```diff
@@ -2204,6 +2204,31 @@ function tests(): Test[] {
         data: ['some', 'array', 1, 2, 3, { object: 'value' }],
       },
     },
+    {
+      name: 'reproduction https://github.com/prisma/prisma/issues/3576',
+      up: `
+CREATE TABLE items (
+  item_id int(10) UNSIGNED NOT NULL,
+  problematic_column mediumint(8) UNSIGNED DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+ALTER TABLE items
+  ADD PRIMARY KEY (item_id);
+
+INSERT INTO items SET item_id = 123, problematic_column = NULL;
+        `,
+      down: `
+        drop table if exists posts items;
+      `,
+      do: async (client) => {
+        const items = await prisma.item.findMany();
+        return itmes
+      },
+      expect: {
```

Comment:
if it's a findMany it will then be an array, should show up in the logs anyway if it works